src/components/LandingPage/PrivateCloud/index.js

```diff
@@ -0,0 +1,361 @@
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/jamiehaywood">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/u/26779712?v=4"
+                                        title="jamiehaywood"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/dependabot-preview[bot]">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/in/2141?v=4"
+                                        title="dependabot-preview[bot]"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/rushabhnagda11">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/u/3235568?v=4"
+                                        title="rushabhnagda11"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/weyert">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/u/7049?v=4"
+                                        title="weyert"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/casio">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/u/29784?v=4"
+                                        title="casio"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                                <a href="https://github.com/Hungsiro506">
+                                    <img
+                                        src="https://avatars.githubusercontent.com/u/10346923?v=4"
+                                        title="Hungsiro506"
+                                        width="50"
+                                        height="50"
+                                    />
+                                </a>
+                            </div>
```

Comment:
[non-blocking] could we make these contributor avatars into a separate component both for reusability but also for keeping this file a bit more concise? It's not something we'll be working on much so I guess it makes sense to remove it from view here.